.github/workflows/pull-request.yaml

```diff
@@ -10,3 +10,62 @@ jobs:
     name: PR
     uses: canonical/observability/.github/workflows/charm-pull-request.yaml@main
     secrets: inherit
+
+  test-against-coordinator:
```

Comment:
Not sure why this is here: can't we do the same using the centralized CI + a itest that deploys the coordinator?

Reply:
I would like to test each PR to the worker against the coordinator's main. a centralized ci would only do that on a schedule. good for gating before releases, but not for individual prs